src/java/org/pantsbuild/tools/junit/impl/ScalaTestUtil.java

```diff
@@ -5,6 +5,20 @@
 public final class ScalaTestUtil {
   private ScalaTestUtil() {}
 
+  /**
+   * org.scalatest.Suite.class, loaded with runtime reflection to avoid the
+   * scalatest dependency.
+   */
+  private static final Class<?> suiteClass;
+  static {
+    try {
+      suiteClass = Class.forName("org.scalatest.Suite");
```

Comment:
If we're doing this for the suiteClass, maybe it'd make sense to cache the runner class in the same way, since it'll be looked up once per scala test.

Reply:
Yeah that makes sense. Any idea if it would make sense to cache the actual `Runner` instance that is returned instead of the `Class<Runner>` that is used to construct it?